src/In.ProjectEKA.OtpService/Notification/Action.cs

```diff
@@ -2,6 +2,6 @@ namespace In.ProjectEKA.OtpService.Notification
 {
     public enum Action
     {
```

Comment:
Don't you feel like we should have test for this?

Reply:
I am not sure... to the NotificationService.SendNotification?

Reply:
Added tests